config/locals.js

```diff
@@ -40,6 +43,44 @@ function cacheCampaign(phoenixCampaign) {
 }
 
 /**
+ * Create Doing/Completed Mobile Commons Groups to support Mobile Commons broadcasting.
+ * @see https://github.com/DoSomething/gambit/issues/673
+ */
+function createMobileCommonsGroupsForCampaign(campaignModel) {
+  const campaignScope = campaignModel;
+  const prefix = `env=${process.env.NODE_ENV} campaign_id=${campaignModel._id}`;
```

Comment:
We'll want to include Campaign Run from https://github.com/DoSomething/phoenix-js/pull/16.

The hope is these Mobile Commons Groups are short-term fix for a more robust subscriber solution for Doing/Completed. But if we're still using this code in a year -- if we want to send a broadcast to all COMPLETED but re-open the campaign, we'll also need to clear the COMPLETED group -- if we add the Current Run ID into the prefix when we create the group, we avoid that scenario.